ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/segments/winter/.content.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@rakeshkumar-verma : why do we need this segment ?

Reply:
deleted